src/Microsoft.Health.Fhir.TemplateManagement.UnitTests/TemplateCollectionProviderFactoryTests.cs

```diff
@@ -76,12 +94,32 @@ public void GivenAWrongDefaultTemplatePath_WhenInitDefaultTemplate_ExceptionWill
         public void GiveNewDefaultTemplateTarGzFile_WhenInitDefaultTemplate_DefaultTemplatesWillBeInit()
         {
             string imageReference = ImageInfo.DefaultTemplateImageReference;
-            CreateTarGz("NewDefaultTemplates.tar.gz", templateFolder);
+            CreateTarGz("NewDefaultTemplates.tar.gz", hl7v2TemplateFolder);
             TemplateCollectionProviderFactory factory = new TemplateCollectionProviderFactory(_cache, Options.Create(_config));
             factory.InitDefaultTemplates("NewDefaultTemplates.tar.gz");
             Assert.NotNull(factory.CreateProvider(imageReference, string.Empty));
         }
 
+        [Fact]
+        public void GiveHl7v2NewDefaultTemplateTarGzFile_WhenInitDefaultTemplate_DefaultTemplatesWillBeInit()
+        {
+            string imageReference = ImageInfo.GetDefaultTemplateImageReferenceByDatatype(DataType.Hl7v2);
+            CreateTarGz("Hl7v2NewDefaultTemplates.tar.gz", hl7v2TemplateFolder);
+            TemplateCollectionProviderFactory factory = new TemplateCollectionProviderFactory(_cache, Options.Create(_config));
```

Comment:
we can consider using constants if use the string frequently

Reply:
good suggestion.